ros-dev-tools/ros-dev-tools.control

```diff
@@ -0,0 +1,15 @@
+Section: misc
+Priority: optional
+Homepage: https://github.com/ros-infrastructure/infra-variants
+Standards-Version: 3.9.2
+
+Package: ros-dev-tools
+Version: 1.0.0
+Maintainer: Steven! Ragnarök <steven@openrobotics.org>
+Depends: ros-build-essential, python3-bloom, python3-colcon-common-extensions, python3-rosdep, python3-vcstool
+Architecture: all
+Copyright: copyright
+Readme: README.md
+Description: Developer Tools for ROS
+ ROS Infrastructure variant including packages generally useful during ROS
+ development.
```

Comment:
Given that this is an end-user facing package, I'm thinking that we shouldn't use "ROS Infrastructure" nor "variant" terminology here.  How about:

```
Metapackage that includes packages generally useful during ROS development.
```

(I'm not wed to the term "metapackage" here, so if you have a different name, that's fine)

Same goes for the description in the spec file below.

Reply:
> Given that this is an end-user facing package, I'm thinking that we shouldn't use "ROS Infrastructure" nor "variant" terminology here. How about:

If you wanted to use the term "metapackage" then we ought not have called this repository "infra-variants". I think that ship has sailed and we should use one term throughout.

```
Variant which includes packages generally useful during ROS development.
```

works for me.

Reply:
That works for me too.

Reply:
Updated in 4ae9783.